src/entity-serializer.ts

```diff
@@ -99,6 +100,10 @@ export class EntitySerializer {
 		this._propertyAliases.set(type, aliases);
 	}
 
+	registerValueResolver(resolver: AsyncValueResolver) {
```

Comment:
`AsyncValueResolvers` are now registered as a global concept for a given model. This avoids the need for users to pass resolver functions when creating entities.